adoptingerlang.org

```diff
@@ -1331,6 +1331,364 @@ Repeat this exercise from time to time, and eventually do some [[https://princip
 :EXPORT_HUGO_MENU: :menu main :parent development
 :END:
 
+One major distinction between programming 20 years ago and programming today is in how software projects shifted in their approaches. Whereas material of yesterday focused a lot on writing reusable and extendable components within each project, the current trend is to make smaller, isolated projects that each individually ends up being easy to throw away and replace. Current microservice implementations and Javascript dependency trees may lead us to believe no project is too small to be discardable.
```

Comment:
"in how software projects shifted in their approaches"

Not sure how to rephrase this but wanted to note it reads funky.